app/_src/gateway/upgrade/index.md

```diff
@@ -67,7 +67,7 @@ between the 2.x and 3.x series (both this version and prior versions) and in the
 open-source (OSS) and Enterprise Gateway [changelogs](/gateway/changelog/). Since {{site.base_gateway}}
 is built on an open-source foundation, any breaking changes in OSS affect all {{site.base_gateway}} packages.
 
-An upgrade path is subject to a wide spectrum of conditions, and there is not a one-size-fits-all way applicable to all customers, depending on the deployment modes, custom plugins, customers’ technical capabilities, hardware capacities, SLA, etc. Our engineers should discuss thoroughly and carefully with customers before taking any action.
+An upgrade path is subject to a wide spectrum of conditions, and there is not a one-size-fits-all way applicable to all customers, depending on the deployment modes, custom plugins, customers’ technical capabilities, hardware capacities, SLA, etc. Our engineers should discuss the upgrade process thoroughly and carefully with you before you take any action.
```

Comment:
```suggestion
An upgrade path is subject to a wide spectrum of conditions, and there is not a one-size-fits-all way applicable to all users, depending on the deployment modes, custom plugins, customers’ technical capabilities, hardware capacities, SLA, etc. You should discuss the upgrade process thoroughly and carefully with our engineers before you take any action.
```
Nice catch. How about this edit, going slightly further? So that it's not implied that our engineers are reaching out to them, but they're reaching out to us.